explorer/src/pages/TransactionDetailsPage.tsx

```diff
@@ -410,6 +415,129 @@ function AccountsCard({
   );
 }
 
+type TokenBalanceRow = {
+  account: PublicKey;
+  mint: string;
+  balance: TokenAmount;
+  delta: number;
+  accountIndex: number;
+};
+
+function generateTokenBalanceRows(
+  preTokenBalances: TokenBalance[],
+  postTokenBalances: TokenBalance[],
+  accounts: ParsedMessageAccount[]
+): TokenBalanceRow[] {
+  let preBalanceMap: { [index: number]: TokenBalance } = {};
+
+  preTokenBalances.forEach(
+    (balance) => (preBalanceMap[balance.accountIndex] = balance)
+  );
+
+  return postTokenBalances
```

Comment:
Will `postTokenBalances` contain tokens that were completely spent during a transaction (post balance of 0)?

Reply:
Yes it will.